### PR TITLE
French translation of the `noreferrer` reference

### PR DESCRIPTION
**Pull request by audrasjb**
This PR creates the French translation of the `noreferrer` reference.

### CODE REVIEW

File: files/fr/web/html/link_types/noreferrer/index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: 'Types de liens : noreferrer'
+slug: Web/HTML/Link_types/noreferrer
+browser-compat: html.elements.a.rel.noreferrer
+translation_of: 'Web/HTML/Link_types/noreferrer'
+---
+<p>Le mot-clé <strong><code>noreferrer</code></strong> de l'attribut <code><a href="/fr/docs/Web/HTML/Attributes/rel">rel</a></code> des éléments {{HTMLElement("a")}}, {{HTMLElement("area")}} et {{HTMLElement("form")}} indique au navigateur, lorsque celui-ci ouvre la ressource cible, d'omettre l'en-tête {{HTTPHeader("Referer")}} et de ne divulguer aucune information de <i lang="en">referrer</i>, comme si le mot-clé <code><a href="/fr/docs/Web/HTML/Link_types/noopener">noopener</a></code> avait également été spécifié (il n'est donc pas utile d'indiquer les deux).</p>
```
Comment (SphinxKnight) [inline files/fr/web/html/link_types/noreferrer/index.html:7]: A slight rewording which I think is necessary, to stick to the English meaning "comme si le mot-clé noopener avait également été spécifié" > "Lorsque noreferrer est indiqué, le navigateur se comporte également comme si le mot-clé noopener avait été spécifié (il n'est donc pas utile...)."

**Review by SphinxKnight — CHANGES_REQUESTED** [review]
A minor rewording which seems more accurate regarding the English meaning.

File: files/fr/web/html/link_types/noreferrer/index.html (hunk 1 above)
Comment (audrasjb) [reply files/fr/web/html/link_types/noreferrer/index.html]: Like in https://github.com/mdn/translated-content/pull/1799/commits/2295008695745b45ca1b02597150fd9bd2d5e809 ?

File: files/fr/web/html/link_types/noreferrer/index.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+title: 'Types de liens : noreferrer'
+slug: Web/HTML/Link_types/noreferrer
+browser-compat: html.elements.a.rel.noreferrer
+translation_of: 'Web/HTML/Link_types/noreferrer'
+---
+<p>Le mot-clé <strong><code>noreferrer</code></strong> de l'attribut <code><a href="/fr/docs/Web/HTML/Attributes/rel">rel</a></code> des éléments {{HTMLElement("a")}}, {{HTMLElement("area")}} et {{HTMLElement("form")}} indique au navigateur, lorsque celui-ci ouvre la ressource cible, d'omettre l'en-tête {{HTTPHeader("Referer")}} et de ne divulguer aucune information de <i lang="en">referrer</i>, comme si le mot-clé <code><a href="/fr/docs/Web/HTML/Link_types/noopener">noopener</a></code> avait également été spécifié. Lorsque <code>noreferrer</code> est indiqué, le navigateur se comporte également comme si le mot-clé <code>noopener</code> avait été spécifié (il n'est donc pas utile d'indiquer les deux).</p>
```
Comment (SphinxKnight) [inline files/fr/web/html/link_types/noreferrer/index.html:7]: well there are now two mentions of noopener ^^ (https://www.youtube.com/watch?v=feHdY7KZP_M&t=4s)
I prefer the second one but I'll let you make your call :)

Comment (audrasjb) [reply files/fr/web/html/link_types/noreferrer/index.html]: loool fixed in https://github.com/mdn/translated-content/pull/1799/commits/7057e75a6e862d24727bb1e6b4f4778c37bc6994 😉

**Review by SphinxKnight — APPROVED** [review]

File: files/fr/web/html/link_types/noreferrer/index.html (hunk 2 above)
Comment (SphinxKnight) [reply files/fr/web/html/link_types/noreferrer/index.html]: Too bad I'm going to squash this commit :D

**Pull request merged**